topology/Langston University/LUCCRE/LUCILLE_downtime.yaml

```
@@ -336,3 +336,27 @@
   Severity: Outage
   StartTime: May 16, 2018 13:00 +0000
 # ----------------------------------------------------------
+- Class: SCHEDULED
+  ID: 235808540
+  Description: Upgrade to EL7
+  Severity: Outage
+  StartTime: Jun 01, 2019 00:00 +0000
+  EndTime: Jul 31, 2019 18:00 +0000
+  CreatedTime: May 22, 2019 22:07 +0000
+  ResourceName: Lucille_CE
+  Services:
+  - CE
+  - GridFtp
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 235809389
+  Description: Upgrade to EL7
+  Severity: Outage
+  StartTime: Jun 01, 2019 00:00 +0000
+  EndTime: Jul 31, 2019 18:00 +0000
+  CreatedTime: May 22, 2019 22:08 +0000
+  ResourceName: LUHEP_SE
```

Fixing up the SE resource name

```suggestion
  ResourceName: Lucille_SE
```